src/api/status/.eslintrc.js

```diff
@@ -1,6 +1,15 @@
 module.exports = {
-  env: {
-    browser: true,
-    node: true,
-  },
+  extends: '@senecadot/eslint-config-telescope',
```

Comment:
`extends: '@senecacdot/eslint-config-telescope',`

missing a `c`

Reply:
I just curious when looking into this PR. It work even with a typo ?

Reply:
I think it knows because the in the `pnpm-lock.yaml` file has the path link to the dependency.

Reply:
@JiaHua-Zou Awesome. Good to know !